src/modules/services/influxdb.nix

```diff
@@ -0,0 +1,22 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.services.influxdb;
+  types = lib.types;
+in
+{
+  options.services.influxdb = {
+    enable = lib.mkEnableOption "influxdb";
+
+    package = lib.mkOption {
+      type = types.package;
+      description = "An open-source distributed time series database";
+      default = pkgs.influxdb;
+      defaultText = lib.literalExpression "pkgs.influxdb";
+    };
```

Comment:
Usually we'd also add `config` to pass the configuration to the daemon

Reply:
I'll try to add a config! It's just kinda hard, since I have no clue of influx nor nix 😅

Reply:
But would the pr otherwise be fine, apart from the missing config?

Reply:
Otherwise looks good! It doesn't have to have anything in the config if it works with the default one well :)

Reply:
I added a config and tried to test for a changed setting (the modified default port) to make sure it is working.

The service seems to need some sort of complete config, not only some lonely changed setting.

I wasn't sure how where to point the directories it needs write access to (`/tmp` atm), that might need to be changed to something making more sense.